public/app/core/crash/index.ts

```diff
@@ -36,6 +37,9 @@ export function initializeCrashDetection() {
     return;
   }
 
+  let lastInteraction = Date.now();
+  document.body.addEventListener('click', () => (lastInteraction = Date.now()));
```

Comment:
Is click enough? Like what about keyboard events? Also I would think using stopPropagation() could prevent even reaching the body (and we use it quite a bit). If this does not have to be super accurate maybe we can just add a comment acknowledging that.

Reply:
Good point on the propagation, but adding `stopPropagation()` would break click events elsewhere? This seems like a first step to filter out false positives, sure adding click events too would cast a wider net.